Dockerfile.mysql

```diff
@@ -0,0 +1,12 @@
+FROM mysql:latest
+
+ENV MYSQL_ROOT_PASSWORD=1234
```

Comment:
Using hardcoded passwords in the Dockerfile is a security risk. Consider using Docker secrets or environment variables passed at runtime to manage sensitive information securely.